refinery/core/api.py

```diff
@@ -498,6 +498,13 @@ class Meta:
         }
 
     def dehydrate(self, bundle):
+        version_details = bundle.obj.get_version_details()
+
+        if version_details is None:
```

Comment:
I don't think its _super_ necessary, but it does cover the edge-case where an importing DataSet is fully created inbetween the calls to `dehydrate` and `obj_get`/`obj_get_list`

This is less of an edge case when we are fetching many hundreds of DataSets